### PR TITLE
Configure CodeClimate

### PR DESCRIPTION
**Pull request by puregarlic**
# Changes introduced

This PR adds a configuration file for CodeClimate for better analysis of our code's technical debt.

## Contributor checklist

- [ ] Wrote/updated tests for introduced changes
- [ ] Added new stories for created/modified components (if applicable)
- [ ] Checked Storybook for Accessibility issues (if applicable)
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [x] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [ ] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] Verified that stories were written/modified (if applicable)
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [ ] Checked out the branch and tested changes locally
- [ ] Run storybook and checked that there are no accessibility issues

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+plugins:
+  fixme:
+    enabled: true
+    config:
+      strings:
```
Comment (tnoelcke) [inline .codeclimate.yml:5]: This doesn't seem right is there something wrong with the codeclimate.yaml configuration in general or should we be adding this as a technical debt issue?

**Review by tnoelcke — CHANGES_REQUESTED** [review]

File: .codeclimate.yml (hunk 1 above)
Comment (puregarlic) [reply .codeclimate.yml:5]: No, this is the intended configuration. `fixme` is the name of a CodeClimate engine that identifies suspicious comments and raises them as technical debt issues.

Comment (tnoelcke) [reply .codeclimate.yml:5]: Ok Makes sense. I'll get this merged. Thanks!

**Review by tnoelcke — APPROVED** [review]

**Pull request merged**